packages/core-bridge/index.d.ts

```diff
@@ -65,6 +65,25 @@ export interface CoreOptions {
    * Options for communicating with the Temporal server
    */
   serverOptions: ServerOptions;
+  /**
+   * Telemetry options
+   */
+  telemetryOptions: TelemetryOptions;
+}
+
+export interface TelemetryOptions {
+  /**
+   * If set, telemetry is turned on and this URL must be the URL of a gRPC
+   * OTel collector.
+   */
+  oTelCollectorUrl?: string
+  /**
+   * A string in the env filter format specified here:
+   * https://docs.rs/tracing-subscriber/0.2.20/tracing_subscriber/struct.EnvFilter.html
+   *
+   * Which determines what tracing data is collected in the Core SDK
+   */
+  tracingFilter?: string
```

Comment:
Is there a way to simplify this filter?
Are there multiple targets that need to be filtered?
It'd be nice if we could just set a log level or some other value that's nicer to use than `temporal_sdk_core=INFO`

Reply:
It's easy enough to add that by just transforming simple CLI options into the appropriate string. Being able to set this gives a lot of control over showing debug output for things like the underlying GRPC library if needed.